Resilio Connect API/JSMonitor/utils.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    findArrayDiff,
+};
+
+// creates a new array that contains the values that the lists do not share
+function findArrayDiff(a1, a2) {
```

Comment:
Would you be able to tell the diff between Agents that were added and Agents that were removed?

Reply:
Maybe if after finding the difference of the arrays, we find the original array of each element in the array we are returning. If an Agent was removed, that element would only be part of the first array we are inputting into the function, and if an agent was added it would only be part of the second array.

Reply:
To make this function useful for our purposes I can see two possible solutions:
(1) limit it to calculate array1 - array2 .  so if array1 is the latest and array2 is an older one, it will return only the ids that are in array1 and not in 2.
(2) a second, more complex solution, would be to return 2 arrays.  one contains array1-array2 and the other returns array2-array1;  a variant of that would be to return an array of objects and each object contains the ID and some value that shows if the ID was missing from 1 or from 2

Reply:
I will for now limit it to calculating array1 - array2. Would you explain the second option more tomorrow during our call? I think an explanation/example would help me understand it better.

Reply:
Because it seems like the first solution would be able to track agents that are added, not ones that are removed.